tests/benchmarks/benchmark_result.py

```diff
@@ -0,0 +1,29 @@
+import json
+from typing import Optional
+
+import attr
+
+from mentat.transcripts import Transcript
+
+
+@attr.define
+class BenchmarkResult:
```

Comment:
I would've preferred to make this a TypedDict if 3.11 was our minimum version (NotRequired was added), but for 3.10 support I think this (with to_json and from_json) is definitely the best call.